webui/cypress/support/commands.js

```diff
@@ -23,3 +23,4 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite("visit", (originalFn, url, options) => { ... })
+import 'cypress-file-upload';
```

Comment:
Missing newline

Reply:
Is this file appropriate for importing modules?

Reply:
This import has been done according to instructions to work with upload module